terraform/cloudfoundry/cf_rds.tf

```diff
@@ -40,7 +40,7 @@ resource "aws_db_instance" "cf" {
   allocated_storage    = 10
   engine               = "postgres"
   engine_version       = "9.5.4"
-  instance_class       = "db.t2.small"
+  instance_class       = "db.m4.large"
```

Comment:
> Could you explain why?

There are a lot of places were we do not worry about the costs. We have plenty of VMs running that could be merged and save costs. There are tenant DB instances running that are either oversized or not being used. 

But this DB can have direct impact on the platform performance and even availability. Why worry so much about the costs when we do not worry so much in other places.

Said that, it is fine, `m3.medium` looks good.